packages/core/src/tracing/instrumentation/database/ioredis.js

```diff
@@ -96,8 +75,15 @@ function instrumentSendCommand(original) {
     return cls.ns.runAndReturn(() => {
       const span = cls.startSpan(exports.spanName, constants.EXIT);
       span.stack = tracingUtil.getStackTrace(wrappedInternalSendCommand);
+
+      const connection = `${client.options.host}:${client.options.port}`;
+
+      // TODO: https://jsw.ibm.com/browse/INSTA-14540
+      // if (client.isCluster) {
+      //  connection = client.startupNodes.map(node => `${node.host}:${node.port}`).join(',');
+
       span.data.redis = {
-        connection: clusterConnectionString || `${client.options.host}:${client.options.port}`,
+        connection,
```

Comment:
qs: What will be the connection string for the cluster?

Reply:
IP Address